index.html

```diff
@@ -13,25 +12,48 @@
     rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
 </head>
-
 <body>
-  <header>
-    <nav class="nav_bar flex">
-      <h1 class="nav_brand">Rapheal Okonta</h1>
-      <ul class="nav_ulist flex">
-        <li class="nav_link"><a href="#recent_works">Portfolio</a></li>
-        <li class="nav_link"><a href="#About">About</a></li>
-        <li class="nav_link"><a href="#form_container_link Contact">Contact</a></li>
-        <li class="nav_link_email"><a href="#"><img src="image folder/Disabled.png" alt="email_icon"></a></li>
-      </ul>
-      <div class="hamburger">
-        <span class="hamburger_icon"></span>
-        <span class="hamburger_icon"></span>
-        <span class="hamburger_icon"></span>
+ <header>
+      <nav class="nav_bar">
+        <a href="#" class="nav_branding">Rapheal Okonta</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#recent_works_link" class="nav_link">Portfolio</a>
+              <hr class="hr-line_break3">
+          </li>
+          <li class="nav_item">
+            <a href="#aboutmeProject" class="nav_link">About </a>
+          <hr class="hr-line_break3">
+                
+          </li>
+          <li class="nav_item">
+            <a href="#form_container_link" class="nav_link">Contact</a>
+            <hr class="hr-line_break3">
+          </li>
+              <li class="nav_link_email"><a href="#"><img src="image folder/Disabled.png" alt="email_icon"></a></li>
+        </ul>
+        <div class="hamburger_icon">
+          <span class="nav_email_icon"></span>
+          <span class="nav_email_icon"></span>
+          <span class="nav_email_icon"></span>
+        </div>
+      </nav>
+       <div class="mobile_nav_container">
+         <ul class="nav_menu2">
+          <li class="nav_item2">
+            <a href="#recent_works_link" class="nav_link">Portfolio</a>
+          </li>
+          <li class="nav_item2">
+            <a href="#aboutmeProject" class="nav_link">About </a>  
+          </li>
+          <li class="nav_item2">
+            <a href="#form_container_link" class="nav_link">Contact</a>
+          
+          </li>
+        
       </div>
```

Comment:
- [ ] As stated the project requirement When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.But In the case of your when I clicked on any of the mobile menu options, it doesn't refer to correct part of page as show with this [video](https://drive.google.com/file/d/1PNp4sFDHTbio3mkH86V_x2Y43AFMYNb3/view?usp=share_link) , Kindly Fix it.